scripts/globals/barge.lua

```diff
@@ -0,0 +1,286 @@
+------------------------------------
+-- Phanauet Channel
+-- https://ffxiclopedia.fandom.com/wiki/Phanauet_Channel
+------------------------------------
+-- Cut Scenes
+------------------------------------
+require("scripts/globals/keyitems")
+require("scripts/globals/zone")
+require("scripts/globals/npc_util")
+local ID = require('scripts/zones/Carpenters_Landing/IDs')
+------------------------------------
+xi = xi or {}
+xi.barge = xi.barge or {}
+
+-- set true for debug output
+local verbose = true
+
+local act =
+{
+    ARRIVE     = 0,
+    DEPART     = 1,
+    ARRIVE_OOS = 2,
+    DEPART_OOS = 3,
+}
+
+-- Barge Destinations
+local dest =
+{
+    SOUTH_LANDING         = 0, -- South Landing via the main canal
```

Comment:
Just curious, but did these also need to start from 1? I remember briefly hearing you talk about this in VC